python/dlr/neologger.py

```diff
@@ -1,12 +1,7 @@
 """Logging API"""
 import logging
 
-'''
-Trusted error logs are mounted by default on the algorithm container's volume. 
-Any logs written to /opt/ml/errors/errors.log will be reported back to sagemaker.
-DO NOT CHANGE THE LOG FILE PATH/NAME
-'''
-def create_logger(log_file="/opt/ml/errors/errors.log", log_level=logging.DEBUG, verbose=True):
+def create_logger(log_file=None, log_level=logging.DEBUG, verbose=True):
```

Comment:
Since log_file is optional, maybe put it as the last parameter.
`verbose` is confusing. . Lets find better name. Maybe `log_to_console=True/False`

Reply:
Done